plugins/USBPrinting/USBPrinterOutputDevice.py

```diff
@@ -96,6 +96,9 @@ def requestWrite(self, nodes, file_name = None, filter_by_machine = False, file_
         if self._is_printing:
             return  # Aleady printing
 
+        # cancel any ongoing preheat timer before starting a print
+        self._printers[0].stopPreheatTimers()
```

Comment:
This should actually be:
`self._printers[0].getController().stopPreheatTimers()`